spec/system/case_contacts/new_spec.rb

```diff
@@ -119,7 +119,9 @@
 
         expect(page).to have_text("Confirm Note Content")
         hello_line = page.body.split("\n").select { |x| x.include?("Hello") }
-        expect(hello_line).to eq(["        <div id=\"note-content\"><h1>Hello world</h1></div>"])
+        # according to app/javascript/src/case_contact.js
+        expect(hello_line)
+          .to eq(["        <div id=\"note-content\">&lt;h1&gt;Hello world&lt;/h1&gt;</div>"])
```

Comment:
This has already been fixed. Maybe delete the changes to prevent conflicts. https://github.com/rubyforgood/casa/pull/2528/files#diff-a96fabcd91219b5fe44cd2638b2acb405090d02620c248fa45a2cff86d78e14e